src/components/20-molecules/image-upload/story.js

```diff
@@ -0,0 +1,142 @@
+  .add('Image upload - default', () => {
+    const sizeSliderOptions = {
+      range: true,
+      min: 1,
+      max: 500,
+      step: 10,
+    };
+    const numberOfFilesSliderOptions = {
+      range: true,
+      min: 1,
+      max: 20,
+      step: 1,
+    };
+    const inputFileText = text('inputFileText', 'Upload file');
+    const maxSizeOfSingleFileKB = number(
+      'maxSizeOfSingleFileKB',
+      100,
+      sizeSliderOptions
+    );
+    const maxSizeOfAllFilesKB = number(
+      'maxSizeOfAllFilesKB',
+      500,
+      sizeSliderOptions
+    );
+    const maxNumberOfFiles = number(
+      'maxNumberOfFiles',
+      10,
+      numberOfFilesSliderOptions
+    );
+    const deleteStatusText = text('deleteStatusText', 'Delete');
+    const addStatusText = text('addStatusText', 'Add more');
+    const fileTooBigStatusText = text(
+      'fileTooBigStatusText',
+      `File size exceeds maximum size`
+    );
+    const filesTooBigStatusText = text(
+      'filesTooBigStatusText',
+      `File sizes exceed maximum size`
+    );
+    const tooManyFilesStatusText = text(
+      'tooManyFilesStatusText',
+      `You exceeded the maximum number of files`
+    );
+    const orText = text('orText', 'or');
+    const infoText = text('infoText', 'Drag and drop to upload your file');
+    const iconOptions = select('icon', icons, 'cloud-upload');
+    const headerText = text('headerText', descriptionText);
+    const wrapper = document.createElement('div');
+
+    const template = html`
+      <div style="width:453px;">
+        <axa-image-upload
+          inputFileText="${inputFileText}"
+          maxSizeOfSingleFileKB="${maxSizeOfSingleFileKB}"
+          maxSizeOfAllFilesKB="${maxSizeOfAllFilesKB}"
+          maxNumberOfFiles="${maxNumberOfFiles}"
+          deleteStatusText="${deleteStatusText}"
+          addStatusText="${addStatusText}"
+          fileTooBigStatusText="${fileTooBigStatusText}"
+          filesTooBigStatusText="${filesTooBigStatusText}"
+          tooManyFilesStatusText="${tooManyFilesStatusText}"
+          orText="${orText}"
+          infoText="${infoText}"
+          icon="${iconOptions}"
+          >${headerText}</axa-image-upload
+        >
+      </div>
+    `;
+
+    render(template, wrapper);
+    return wrapper;
+  })
+  .add('Image upload - maxSizeOfSingleFileKB', () => {
+    const wrapper = document.createElement('div');
+    const template = html`
+      <div style="width:453px;">
+        <axa-image-upload maxSizeOfSingleFileKB="1"
+          >${descriptionText}</axa-image-upload
+        >
+      </div>
+    `;
+
+    render(template, wrapper);
+    return wrapper;
+  })
+  .add('Image upload - maxNumberOfFiles', () => {
+    const wrapper = document.createElement('div');
+    const template = html`
+      <div style="width:453px;">
+        <axa-image-upload maxNumberOfFiles="1"
+          >${descriptionText}</axa-image-upload
+        >
+      </div>
+    `;
+
+    render(template, wrapper);
```

Comment:
I'm using knobs, but I used the stories for testing. Therefore I can use `fixture()`. How would you do that with knobs?

Reply:
Daniel, Marco and me decided, that we use knobs links to test, instead of spamming demos into our production code, that only reason to exist is to make the component testable. We should NEVER write code that is being accessed to the public, that is only meant to run some tests.

Reply:
How does this works "knobs links to test"? Can you tell me a component where this is already implemented?